### PR TITLE
Dont transform classes

### PR DESCRIPTION
**Pull request by koddsson**
Reduce the size of the distribution by not transforming the `class` keyword. Dropped IE11 support in the meanwhile and updated some packages.

### CODE REVIEW

**Comment by koddsson** [comment]
Looking at that test failure now 👀

File: emoji-detection.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-const onLinux = /\bLinux\b/.test(navigator.userAgent)
-const onFreeBSD = /\bFreeBSD\b/.test(navigator.userAgent)
-
 export function isEmojiSupported(): boolean {
+  const onWindows7 = /\bWindows NT 6.1\b/.test(navigator.userAgent)
+  const onWindows8 = /\bWindows NT 6.2\b/.test(navigator.userAgent)
+  const onWindows81 = /\bWindows NT 6.3\b/.test(navigator.userAgent)
+  const onFreeBSD = /\bFreeBSD\b/.test(navigator.userAgent)
+  const onLinux = /\bLinux\b/.test(navigator.userAgent)
+
```
Comment (keithamus) [inline emoji-detection.js]: Why have these been moved? Is this to make testing simpler?

Comment (koddsson) [reply emoji-detection.js]: Yeah, it tests the `navigator.userAgent` string when the script parsers but then in the tests we override it to test different browsers. I'm not sure if there is a better way to do this but I did this essentially to make sure that we don't read the `navigator.userAgent` property until the tests are done mocking it.

Comment (keithamus) [reply emoji-detection.js]: 👍 all is good

**Pull request merged**